### PR TITLE
Fix grid missing lines. Fixes: #20246

### LINKED ISSUE #20246: Grid missing lines
**Godot version:**
Godot Engine v3.1.dev.calinou.a3f1ee5

**OS/device including version:**
Win 10 64bit

**Issue description:**
When view origin is turned off in the view menu, grid doesn't draw lines where x and z origin lines would otherwise be.

capture

### PR DESCRIPTION
**Pull request by Douglas-W-Williams**
Fixes #20246

### CODE REVIEW

**Comment by Zephilinox** [comment]
Wouldn't this cause those lines to be drawn when view origin is turned on as well?

**Comment by ordigdug** [comment]
@Zephilinox Yes, but the origin draws on top of the grid.

File: editor/plugins/spatial_editor_plugin.cpp [hunk 1]
```diff
@@ -4641,9 +4641,7 @@ void SpatialEditor::_init_grid() {
 			Vector3 p2_dest = p2 * (-axis_n1 + axis_n2);
 
 			Color line_color = secondary_grid_color;
-			if (j == 0) {
-				continue;
-			} else if (j % primary_grid_steps == 0) {
+			if (j == 0 || j % primary_grid_steps == 0) {
```
Comment (akien-mga) [inline editor/plugins/spatial_editor_plugin.cpp]: You can remove the first condition, if `j == 0` then the second condition is true by definition.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**